cmd/dockerd/config.go

```diff
@@ -53,6 +55,8 @@ func installCommonConfigFlags(conf *config.Config, flags *pflag.FlagSet) {
 
 	flags.StringVar(&conf.MetricsAddress, "metrics-addr", "", "Set default address and port to serve the metrics api on")
 
+	flags.Var(opts.NewQuotedString(&conf.TrustKeyPath), "key-path", "Path to key file for ID and image signing")
```

Comment:
In that case could we make the name `deprecated-key-path`, and Also add 

```go
flags.MarkHidden("deprecated-key-path")
```

So we hide it from help. 

If we expect it to be removed at some point we could use ` MarkDeprecated()` instead of hidden